docs/source/history.rst

```diff
@@ -6,6 +6,16 @@
 
    unmaintained
 
+7.3.3
```

Comment:
This is fine, but you can also just put "Next" so you don't have to guess at the next version number.